### DISCUSSION TITLE
Trouble with Monika's birthday

### DISCUSSION THREAD

**Discussion by ThatOneUsername343** [Monika-After-Story/MonikaModDev#7973] [general]
I'm not sure what I'm supposed to do for Moni's birthday. I know I'm supposed to get cake, balloons, a banner, all that, but idk how to do it and the guides I have seen have confused me. They talk about some sort of file I should have received, but no such file ever showed up in my character folder. 

What do?

**Comment by lunulae** [comment]
Couple of things to help us determine the cause of your problem:
1. Did you visit Monika during the days prior to her birthday? The file for setting up the party shows up within the week before as a random topic.
2. Have you ever time travelled? This messes up in-game events in a big way.

**Comment by ThatOneUsername343** [comment]
I think I time travelled backwards one time, and I only travelled backwards 1 day before resetting the time. And no I haven't visited her for several days.

**Comment by lunulae** [comment]
That'll do it--time travel very specifically messes up the game's ability to run holidays and birthdays. You'll need to reload a persistent from before you time travelled, if you have one. Unfortunately, that is the only possible fix for this issue.

**Comment by ThatOneUsername343** [comment]
That was well over a year ago so I don't really think that's a good idea. Is there any way I can get all her birthday stuff without that file?

**Comment by ThePotatoGuy** [comment]
>And no I haven't visited her for several days.

You mentioned this so I think next time you should visit within the week. 

I doubt time traveling was the problem here. Time traveling can cause lots of issues, but within one day will usually be fine because we have some handling for timezones.
